src/app/product-detail/product-detail.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="card mb-3 top-100 mx-auto" *ngIf="selectedProduct" style="max-width: 640px;">
```

Comment:
No inline styles please.

Reply:
ok :) It is a bootstrap card. I removed the inline style